### DISCUSSION TITLE
Drag&Drop sort objects

### DISCUSSION THREAD

**Discussion by valentin-** [pimcore/pimcore#13441] [Q&A / Support]
Hi, when trying to drag&drop sort objects with manual sorting in (paginated) folders, Pimcore now advices to use the Search & Move feature. I don't understand the concept of moving objects to a specific position/order using this feature. Can someone explain this to me? (Pimcore 10.5).

Example: I want to move an object inside a folder from position 10 to position 2.

![image](https://user-images.githubusercontent.com/90033/197500701-76460d53-feb2-4a2e-840d-a52ed527d136.png)

**Comment by valentin-** [comment]
Hi @dvesh3, could you please explain how sorting works now? D&D sorting has been removed in https://github.com/pimcore/pimcore/issues/12597

Thanks!

**Comment by dvesh3** [comment]
@valentin- basically we removed the possibility to dragndrop elements on manually indexed tree as this could lead to performance issues, if you have thousands of objects on the same level (as this re-indexes the tree elements on every dragndrop) and moreover it was not possible to move elements between pages, so it didn't work properly. For enabling dragndrop, you have to opt for auto indexing on paginated trees. 

please follow the complete discussion here https://github.com/pimcore/pimcore/pull/12555
